BE/app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Traits\ApiResponse;
+use App\Http\Repositories\Auth\AuthActionsRepositoryInterface;
+use App\Http\Requests\LoginRequest;
+use App\Models\User;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Auth;
+
+/**
+ * Class AuthController
+ * @package App\Http\Controllers
+ */
+class AuthController extends Controller
```

Comment:
Because for this task I used Laravel,and Laravel provides an effective authentication system for SPAs , mobile applications, and simple, token based APIs.

Reply:
Sorry, I don't understand. Laravel Sanctum is the recommended auth package for SPAs, but you've used a custom one. Am I missing something?

Reply:
I re-read the very first question, I meant that I did not use custom auth solution, I just created custom controller for login and logout api action, for generated(login), and delete(logout), but all processes in this actions base on Sanctum, and if you check api.php, you can see that there is a auth:sanctum middleware for cover the routes that require authorization